pkg/loop/cmd/loopinstall/main.go

```diff
@@ -0,0 +1,133 @@
+
+	// Process each config file
+	for _, configFile := range flag.Args() {
+		tasks, err := processConfigFile(configFile, verbose)
+		if err != nil {
+			log.Fatalf("Failed to process config file %s: %v", configFile, err)
+		}
+
+		// Check for duplicates across files
+		for _, task := range tasks {
+			key := pluginKey(task.PluginType, task.Plugin)
+			if prevFile, exists := seenPlugins[key]; exists {
+				log.Fatalf("Duplicate plugin found: %s in files %s and %s",
+					key, prevFile, configFile)
+			}
+			seenPlugins[key] = configFile
+		}
+
+		allTasks = append(allTasks, tasks...)
+	}
+
+	// Install all plugins
+	if err := installPlugins(allTasks, concurrency, verbose, outputFile); err != nil {
+		os.Exit(1)
+	}
+}
+
+// printHelp prints usage and help information
+func printHelp() {
+	fmt.Println("Chainlink Plugin Installer")
+	fmt.Println("\nA tool for installing Chainlink plugins from YAML configuration files.")
+	fmt.Println("\nUsage:")
+	fmt.Println("  loopinstall [options] <plugin-config-file> [<plugin-config-file>...]")
+	fmt.Println("\nOptions:")
+	fmt.Println("  -h, --help                Show this help message")
+	fmt.Println("  -v, --verbose             Enable verbose output")
+	fmt.Println("  -c, --concurrency <num>   Set maximum number of concurrent installations (default: 5)")
+	fmt.Println("  -s, --sequential          Install plugins sequentially (no concurrency)")
+	fmt.Println("  -o, --output-installation-artifacts <file>  Path for installation artifacts JSON file")
+	fmt.Println("                             (optional, no installation artifacts written if not specified)")
+	fmt.Println("\nExamples:")
+	fmt.Println("  # Install plugins from the default configuration")
+	fmt.Println("  loopinstall plugins.default.yaml")
+	fmt.Println("")
+	fmt.Println("  # Install plugins with custom installation artifacts filename")
+	fmt.Println("  loopinstall -o ./installation-artifacts.json plugins.default.yaml")
+	fmt.Println("")
+	fmt.Println("  # Install plugins sequentially")
+	fmt.Println("  loopinstall -s plugins.default.yaml")
+	fmt.Println("")
+	fmt.Println("  # Install plugins with environment variable overrides")
+	fmt.Println("  CL_PLUGIN_GOFLAGS=\"-ldflags='-s'\" loopinstall plugins.default.yaml")
+	fmt.Println("")
+	fmt.Println("Environment Variables:")
+	fmt.Println("  CL_PLUGIN_GOFLAGS  Override the goflags option from the configuration")
+	fmt.Println("")
+	fmt.Println("Plugin Configuration Format:")
+	fmt.Println("  defaults:")
+	fmt.Println("    goflags: \"\"     # Default Go build flags")
+	fmt.Println("")
+	fmt.Println("  plugins:")
+	fmt.Println("    plugin-type:")
+	fmt.Println("      - moduleURI: \"github.com/example/module\"")
+	fmt.Println("        gitRef: \"v1.0.0\"")
+	fmt.Println("        installPath: \"./cmd/example\"")
+	fmt.Println("        libs: [\"/path/to/libs/*.so\"]  # Optional library paths (can include glob patterns)")
+	fmt.Println("        # enabled: false  # Optional, defaults to true if omitted")
```

Comment:
nit/ this would b easier to read/edit quoted with `

Reply:
Will get a fix in 👍

Reply:
https://github.com/smartcontractkit/chainlink-common/pull/1138